### PR TITLE
Added documentation for azure api management

### PR DESCRIPTION
**Pull request by sachin-sumologic**
## Purpose of this pull request

This pull request... 

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

[SUMO-250894](https://sumologic.atlassian.net/browse/SUMO-250894)

### CODE REVIEW

File: docs/integrations/microsoft-azure/azure-api-management.md [hunk 1]
```diff
@@ -27,29 +29,181 @@ Azure service sends monitoring data to Azure Monitor, which can then [stream dat
...
+   | parse regex field=resourceId "/SUBSCRIPTIONS/(?<subscription_id>[^/]+)" nodrop
+   | parse field=resourceId "/RESOURCEGROUPS/*/" as resource_group nodrop
+   | parse regex field=resourceId "/PROVIDERS/(?<provider_name>[^/]+)" nodrop
+   | parse regex field=resourceId "/PROVIDERS/[^/]+(?:/LOCATIONS/[^/]+)?/(?<resource_type>[^/]+)/(?<resource_name>.+)" nodrop
+   | parse regex field=resource_name "(?<parent_resource_name>[^/]+)(?:/PROVIDERS/[^/]+)?/(?<service_type>[^/]+)/?(?<service_name>.+)" nodrop
+   | if (isBlank(parent_resource_name), resource_name, parent_resource_name) as resource_name
+   | fields subscription_id, location, provider_name, resource_group, resource_type, resource_name, service_type, service_name
+   ```
+
+### Configure metric rules
```
Comment (himanshu219) [inline docs/integrations/microsoft-azure/azure-api-management.md:89]: we need another metric rule which extracts service "Azure Observability Metadata Extraction Service Level" https://help.sumologic.com/docs/integrations/microsoft-azure/azure-storage/#configure-metric-rules

File: docs/integrations/microsoft-azure/azure-api-management.md [hunk 2]
```diff
@@ -27,29 +29,181 @@ Azure service sends monitoring data to Azure Monitor, which can then [stream dat
...
 
-1. [Configure an HTTP Source](/docs/send-data/collect-from-other-data-sources/azure-monitoring/collect-metrics-azure-monitor/#step-1-configure-an-http-source).
-2. [Configure and deploy the ARM Template](/docs/send-data/collect-from-other-data-sources/azure-monitoring/collect-metrics-azure-monitor/#step-2-configure-azure-resources-using-arm-template).
-3. [Export metrics to Event Hub](/docs/send-data/collect-from-other-data-sources/azure-monitoring/collect-metrics-azure-monitor/#step-3-export-metrics-for-a-particular-resource-to-event-hub). Perform below steps for each Azure API Management service that you want to monitor.
+1. Create hosted collector and tag tenant_name field. <br/><img src={useBaseUrl('img/integrations/microsoft-azure/Azure-Storage-Tag-Tenant-Name.png')} alt="Azure Tag Tenant Name" style={{border: '1px solid gray'}} width="500" />
+2. [Configure an HTTP Source](/docs/send-data/collect-from-other-data-sources/azure-monitoring/collect-metrics-azure-monitor/#step-1-configure-an-http-source).
```
Comment (himanshu219) [inline docs/integrations/microsoft-azure/azure-api-management.md:115]: lets use 1. everywhere

File: docs/integrations/microsoft-azure/azure-api-management.md [hunk 3]
```diff
@@ -27,29 +29,181 @@ Azure service sends monitoring data to Azure Monitor, which can then [stream dat
...
+   1. Use the Event Hub namespace and Event Hub name configured in the previous step in the destination details section. You can use the default policy `RootManageSharedAccessKey` as the policy name.<br/><img src={useBaseUrl('img/send-data/azure-apimanagement-logs.png')} alt="Azure API Management logs" style={{border: '1px solid gray'}} width="800" />
+1. Tag the location field in the source with right location value. <br/><img src={useBaseUrl('img/integrations/microsoft-azure/Azure-Storage-Tag-Location.png')} alt="Azure API Management Tag Location" style={{border: '1px solid gray'}} width="400" />
+
+#### Activity Logs
+
+To collect activity logs, follow the instructions [here](/docs/integrations/microsoft-azure/audit). Do not perform this step in case you are already collecting activity logs for a subscription.
+
+:::note
+Since this source contains logs from multiple regions, make sure that you do not tag this source with the location tag.
```
Comment (himanshu219) [inline docs/integrations/microsoft-azure/azure-api-management.md:143]: instructions for enabling cloud defender ?
Also instructions for recommendation alert ?

File: docs/integrations/microsoft-azure/azure-api-management.md [hunk 4]
```diff
@@ -27,29 +29,181 @@ Azure service sends monitoring data to Azure Monitor, which can then [stream dat
...
+
+### Policy and Recommendations
+The **Azure API Management - Policy and Recommendations** dashboard provides details like Total Recommendation Events, Total Success Policy Events, Total Failed Policy Events, Failed Policy Events, Recent Recommendation Events, Recommendation, Policy etc.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/AzureApiManagement/Azure-API-Management-Policy-and-Recommendations.png')} alt="Azure Database for PostgreSql - Overview" style={{border: '1px solid gray'}} width="800" />
+
+### Subscriptions
+The **Azure API Management - Subscriptions** dashboard provides details like Total Requests by Subscription, Failed Requests by Subscription, Requests by Subscription, Failed Requests by Subscription, Top 10 Failed Subscription, Recent Changes in Subscription etc.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/AzureApiManagement/Azure-API-Management-Subscriptions.png')} alt="Azure Database for PostgreSql - Overview" style={{border: '1px solid gray'}} width="800" />
```
Comment (himanshu219) [inline docs/integrations/microsoft-azure/azure-api-management.md:193]: Remove "Azure Database for PostgreSql" from every alt text for image

**Review by himanshu219 — CHANGES_REQUESTED** [review]
minor changes requested

File: docs/integrations/microsoft-azure/azure-api-management.md [hunk 5]
```diff
@@ -172,6 +172,7 @@ Since this source contains logs from multiple regions, make sure that you do not
 
 Now that you have set up data collection, install the Azure API Management Sumo Logic app to use the pre-configured dashboards that provide visibility into your environment for real-time analysis of overall usage.
 
+import AppInstallNoDataSourceV2 from '../../reuse/apps/app-install-index-apps-v2.md';
```
Comment (himanshu219) [inline docs/integrations/microsoft-azure/azure-api-management.md:175]: why importing  AppInstallNoDataSourceV2 multiple times?

File: docs/integrations/microsoft-azure/azure-api-management.md [hunk 6]
```diff
@@ -14,9 +14,9 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
-* **WebSocket Connection Logs**. To learn more about the resource log schema for Azure API Management, refer to the [Azure documentation](https://learn.microsoft.com/en-us/azure/azure-monitor/reference/tables/apimanagementwebsocketconnectionlogs). The Consumption tier doesn't support the collection of resource logs.
-* **Developer Portal Audit Logs**. To learn more about the resource log schema for Azure API Management, refer to the [Azure documentation](https://learn.microsoft.com/en-us/azure/api-management/developer-portal-audit-log-schema-reference). The Consumption tier doesn't support the collection of resource logs.
+* **Gateway Logs**. Resource Logs for Azure API Management Gateway. To learn more about the resource log schema for Azure API Management, refer to the [Azure documentation](https://learn.microsoft.com/en-us/azure/api-management/gateway-log-schema-reference). The Consumption tier doesn't support the collection of resource logs.
```
Comment (himanshu219) [inline docs/integrations/microsoft-azure/azure-api-management.md:17]: To learn more about the resource log schema for Azure API Management
-> To learn more about its schema, refer to the Azure documentation

File: docs/integrations/microsoft-azure/azure-api-management.md (hunk 5 above)
Comment (sachin-sumologic) [reply docs/integrations/microsoft-azure/azure-api-management.md:175]: Happened by mistakenly, removed.

**Review by himanshu219 — APPROVED** [review]

File: docs/integrations/microsoft-azure/azure-api-management.md [hunk 7]
```diff
@@ -28,28 +30,218 @@ You must explicitly enable diagnostic settings for each Azure API Management ser
...
+
+To collect activity logs, follow the instructions [here](/docs/integrations/microsoft-azure/audit). Do not perform this step in case you are already collecting activity logs for a subscription.
+
+:::note
+Since this source contains logs from multiple regions, make sure that you do not tag this source with the location tag.
+:::
+
+##### Enabling Microsoft Defender for Cloud
+
+For Security events, make sure you enable [Microsoft Defender for Cloud](https://learn.microsoft.com/en-us/azure/defender-for-cloud/tutorial-enable-app-service-plan#enable-the-defender-for-app-service-plan). In Defender Plans Settings page toggle the App Service status under Cloud Workload Protection section.
```
Comment (JV0812) [inline docs/integrations/microsoft-azure/azure-api-management.md:172]:
```suggestion
For Security events, make sure you enable [Microsoft Defender for Cloud](https://learn.microsoft.com/en-us/azure/defender-for-cloud/tutorial-enable-app-service-plan#enable-the-defender-for-app-service-plan). In Defender Plans Settings page toggle the **APIs** status under **Cloud Workload Protection (CWP)** section.
```

**Review by JV0812 — APPROVED** [review]
LGTM

**Review by amee-sumo — APPROVED** [review]
LGTM

**Pull request merged**